chain.schema.json

```
@@ -12,8 +14,12 @@
     "tendermint_version": "0.34",
     "cosmwasm_version": "0.28",
     "cosmwasm_enabled": true,
+    "ibc_go_version": "3.0.0",
```

```suggestion
    "ibc_go_version": "3.3.0",
```

Minor versions are state breaks in IBC :/